src/web/handlers/IGestureHandler.ts

```diff
@@ -7,19 +7,25 @@ import type { GestureHandlerDelegate } from '../tools/GestureHandlerDelegate';
 import type PointerTracker from '../tools/PointerTracker';
 
 export default interface IGestureHandler {
-  getTag: () => number;
-  getState: () => State;
-  getConfig: () => Config;
-  getDelegate: () => GestureHandlerDelegate<unknown, this>;
+  active: boolean;
+  activationIndex: number;
+  awaiting: boolean;
+  handlerTag: number;
+  config: Config;
+  delegate: GestureHandlerDelegate<unknown, this>;
+  tracker: PointerTracker;
+  state: State;
+  shouldCancelWhenOutside: boolean;
+  shouldResetProgress: boolean;
+  enabled: boolean;
+  pointerType: PointerType;
```

Comment:
Shouldn't some of those be marked as `readonly`?

Reply:
Done in [4275e16](https://github.com/software-mansion/react-native-gesture-handler/pull/3263/commits/4275e16d4199fbaad768bd6886e6977010a8b267)

Reply:
Also, let me know what di you think about `protected setters`

Reply:
I don't have a strong opinion of them. On the one hand, they don't do anything at the moment, but on the other the code looks a bit nicer without `_` and if we ever need to add any side-effects to the setters it would require less changed. The question is whether it's likely that we would be adding any side-effects to those setters.

Reply:
While it doesn't seem like we will add side-effects to them, I think that we can leave it for now 😅 Especially that

> the code looks a bit nicer without `_`